src/TwilioChannel.php

```diff
@@ -79,17 +84,22 @@ public function send($notifiable, Notification $notification)
      * Get the address to send a notification to.
      *
      * @param mixed $notifiable
+     * @param Notification $notification
+     * @param TwilioMessage $message
      *
      * @return mixed
      * @throws CouldNotSendNotification
      */
-    protected function getTo($notifiable)
+    protected function getTo($notifiable, $notification, $message)
     {
-        if ($notifiable->routeNotificationFor(self::class)) {
-            return $notifiable->routeNotificationFor(self::class);
+        if ($message->getTo()) {
+            return $message->getTo();
```

Comment:
Sorry, little confused here. Won't this mean that the actual notifiable recipient will be ignored here?

Reply:
This is just to check if any recipient number is provided in the message itself. If provided, this means the developer wants to override the notifiable user's phone number.
And if not overridden, then we can safely use the notifiable's phone number as it was before in the next lines.